public/blog/crosswalk-app-tools-alpha.md

```diff
@@ -0,0 +1,70 @@
+We are happy to announce the alpha version for Crosswalk-app-tools. This is a preview milestone of our forthcoming Node.js based tools for Android app development. Deployment in a production environment is not encouraged at this time.
+
+Crosswalk-app-tools is licensed under the Apache License Version 2.0, please refer to the file LICENSE-APACHE-V2 included in the package.
+
+### Installation
+
+Prerequisites are functional
+
+  1. Android SDK
+  2. Apache Ant
+  3. Node.js
+
+on the system.
+
+In order to get the `crosswalk-app` script available everywhere, global npm installation is required. On most Linux distributions this can be achieved by using `sudo`.
+```
+sudo npm install -g crosswalk-app-tools
+```
+
+### Usage
+
+```
+Crosswalk Application Project and Packaging Tool
+
+    crosswalk-app create <package-id>       Create project <package-id>
+                         --crosswalk=<path> Use downloaded Crosswalk
+                         --channel=<name>   Release channel: stable|beta|canary
+
+    crosswalk-app build [release|debug]     Build project to create packages
+                                            Defaults to debug when not given
+
+    crosswalk-app update <channel>|<version>    Update Crosswalk to latest in named
+                                                channel, or specific version
+
+    crosswalk-app help              Display usage information
+
+    crosswalk-app version           Display version information
+```
+#### Example: Create App
+`crosswalk-app create com.example.foo`: This sets up a skeleton project in directory com.example.foo/, downloads and imports Crosswalk, and puts a sample "hello world" web app under com.example.foo/app/.
+
+#### Example: Build App
+`cd com.example.foo` and then `crosswalk-app build` builds packages. The APKs can be found under pkg/ when done.
+
+For more details, see the project's [README file](https://github.com/crosswalk-project/crosswalk-app-tools/blob/master/README.md).
+
+## Highlights
+
+* Straightforward creation and building of app packages (APKs) for Android. Crosswalk is downloaded and integrated into the build process without user intervention.
+
+* Distributed via the Node Package Manager, with the Android SDK and Ant as only requirements, relying only on standard web app and Android development tools.
+
+* Extensible architecture allows for adding operating system support and package formats over time.
+
```

Comment:
I am skipping this request because the automatic downloading is what makes it most straightforward.